modules/registry-configuring-registry-storage-rhodf-cephrgw.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-vsphere.adoc
+//
+// * registry/configuring_registry_storage/configuring-registry-storage-baremetal.adoc
+//
+// * registry/configuring_registry_storage/Configuring-the-registry-for-rhodf.adoc
+
+:_content-type: PROCEDURE
+[id="registry-configuring-registry-storage-rhodf-cephrgw_{context}"]
+= Configuring the Image Registry Operator to use Ceph RGW storage with Red Hat OpenShift Data Foundation
+
+{rh-storage-first} integrates multiple storage types that you can use with the internal image registry:
+
+* Ceph, a shared and distributed file system and on-premises object storage
+* NooBaa, providing a Multicloud Object Gateway
+
+This procedure outlines the procedure to  configure the image registry to use Ceph RGW storage.
+
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have access to the {product-title} web console.
+* You installed the `oc` CLI.
+* You installed the link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/managing_and_allocating_storage_resources/index#configuring-image-registry-to-use-openshift-data-foundation_rhodf[{rh-storage-first} Operator] to provide object storage and Ceph RGW file storage.
```

Comment:
How about replace the link with https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10 ?

And s/provide object storage and Ceph RGW file storage/Ceph RGW object storage